src/Analyzer.Types/ConfigurationDefinition.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Templates.Analyzer.Types
+{
+    /// <summary>
+    /// Represents a configuration file written in JSON.
+    /// </summary>
+    public class ConfigurationDefinition
```

Comment:
Why isn't severityOverride defined here too?

Reply:
Because it is not yet implemented.

Reply:
Is there a reason why we keep that information in the documentation anyways? I vote for removing it, or at least adding an asterisk also in line 13 to say that it is not supported

Reply:
added. Leaving in doc as it was part of the design